app/controllers/garage/docs/resources_controller.rb

```diff
@@ -2,6 +2,8 @@
 require 'http_accept_language'
 
 class Garage::Docs::ResourcesController < Garage::ApplicationController
+  include HttpAcceptLanguage::EasyAccess
```

Comment:
I don't see detail yet, but is this including needed? http_accept_language's README does not indecates this and seems AC::Base already includes this one. https://github.com/iain/http_accept_language/blob/master/lib%2Fhttp_accept_language%2Frailtie.rb#L7